docs/program-setup/runtime/Execution.md

```diff
@@ -1,7 +1,34 @@
+When writing an Mbed OS program, in whichever development tool you use, you may need to:
+
+* Set up a connection to your board so you can flash and debug your program.
+    * You won't usually need a driver for Mbed enabled boards
+    * Mbed CLI has a built-in serial terminal to display output from your application
```

Comment:
What does that mean? That I don't need to install a terminal app?

Reply:
Yes you can use ```mbed sterm``` as described further in the docs.